### PR TITLE
Fix Logfmt escaping

### LINKED ISSUE #511: LogfmtRenderer  has a bug with escaped double quotes
In the code of LogFmtRenderer there is:

`value = f"{value}".replace('"', '\\"')`

However if you are trying to render an escaped double quote like
`"I want to render this \"string\" with logfmtrenderer"`

it ends up rendering it as
`\"I want to render this \\"string\\" with logfmtrenderer\"`

`\\"` is wrong and won't be parsed correctly by a logfmtparser.

### PR DESCRIPTION
**Pull request by hynek**
Fixes #511, closes #513

@davellas and @madjar – how does this look?

### CODE REVIEW

File: tests/processors/test_renderers.py [hunk 1]
```diff
@@ -275,6 +275,33 @@ def test_newline_in_value(self):
 
         assert r"with_newline=some\nvalue" == rv
 
+    @pytest.mark.parametrize(
+        ("raw", "escaped"),
+        [
+            # Slash by itself does not need to be escaped.
+            (r"a\slash", r"a\slash"),
+            # A quote requires quoting, and escaping the quote.
+            ('a"quote', r"a\"quote"),
```
Comment (madjar) [inline tests/processors/test_renderers.py:284]: This one's not right: if there is a quote in the string, then the output needs to be between quotes, so:
```suggestion
            ('a"quote', r'"a\"quote"'),
```

That'll change the implementation, I guess.

**Review by madjar — CHANGES_REQUESTED** [review]
I marked one of the test cases as not correct. (Checked in loki)

File: tests/processors/test_renderers.py (hunk 1 above)
Comment (hynek) [reply tests/processors/test_renderers.py]: ugh thanks this stuff is so hard to enter

**Comment by hynek** [comment]
Are we there now? 😅

**Review by madjar — APPROVED** [review]
Yep, that looks like what we discussed.

Yay!

**Pull request merged**